traits/trait_types.py

```diff
@@ -3141,7 +3156,7 @@ class Map(TraitType):
 
         The following example defines a ``Person`` class::
 
-            >>> class Person(HasTraits):
+            >>>     erson(HasTraits):
```

Comment:
Looks like I fat-fingered this line.

```suggestion
            >>> class Person(HasTraits):
```